### PR TITLE
Fix(eos_cli_config_gen): Relax schema for empty prefix-lists

### LINKED ISSUE #2596: Feat(eos_cli_config_gen): Relax schema for empty prefix-lists
### Enhancement summary

Empty prefix lists are accepted in EOS configuration and rendered by `eos_cli_config_gen`, even if no sequence list is provided. I need to configure empty prefix-lists as placeholders sometimes when I expect these to be user-defined at a later point in time, to avoid having undefined references in the configuration. However, the schema validation throws warnings:

```
[WARNING]: b"[fabric-switch]: 'Validation Error: prefix_lists[4]': Required
key 'sequence_numbers' is not set in dict."
```

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

Need to define empty prefix-lists in some cases

### Describe the solution you would like

Relax JSON schema validation to accept prefix-list definitions without sequence list. It is accepted and rendered by the `eos_cli_config_gen` templates.

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ghost**
## Change Summary

Relax schema for empty prefix-lists

## Related Issue(s)

Fixes #2596 

## Component(s) name

`arista.avd.eos_cli_config_gen`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/prefix-lists.j2 [hunk 1]
```diff
@@ -2,8 +2,8 @@
 {% for prefix_list in prefix_lists | arista.avd.natural_sort('name') %}
 !
 ip prefix-list {{ prefix_list.name }}
-{%     for sequence in prefix_list.sequence_numbers | arista.avd.natural_sort('sequence') %}
-{%         if sequence.action is arista.avd.defined %}
+{%     for sequence in prefix_list.sequence_numbers | arista.avd.default([]) | arista.avd.natural_sort('sequence') %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/prefix-lists.j2]: Natural sort already defaults to an empty list, so you can skip it where you have sorting.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/prefix_lists.schema.yml [hunk 2]
```diff
@@ -27,13 +25,11 @@ keys:
             keys:
               sequence:
                 type: int
-                required: true
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/prefix_lists.schema.yml]: I don't think we should change the required fields for each list item, so I would revert the changes concerning `sequence` and `action`

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/prefix-lists.j2 (hunk 1 above)
Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/prefix-lists.j2]: fixed!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/prefix_lists.schema.yml (hunk 2 above)
Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/prefix_lists.schema.yml]: Fixed

**Review by ClausHolbechArista — COMMENTED** [review]
We should add a test of this somewhere, so we see that we can in fact generate config for a prefix-list without any sequence numbers.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by carlbuchmann — APPROVED** [review]
LGTM
